jdbc/src/main/java/org/apache/metamodel/jdbc/dialects/SQLServerQueryRewriter.java

```diff
@@ -116,6 +116,14 @@ public String rewriteFilterItem(FilterItem item) {
                 final String dateTimeValue = "CAST('" + format.format(date) + "' AS DATETIME)";
 
                 sb.append(dateTimeValue);
+
+                //Remove TIMESTAMP token as SQL Server doesn't support it
+                int timestampIndex = sb.lastIndexOf("TIMESTAMP");
```

Comment:
I can confirm that a TIMESTAMP is added when using this kind of query:
```Java
final Table dateTest = strategy.getDefaultSchema().getTableByName("datetest");
final Column created = dateTest.getColumnByName("created");
final Column name = dateTest.getColumnByName("name");
Query q = new Query().select(name).from(dateTest).where(created, OperatorType.GREATER_THAN, "2015-01-01);
```

Retrieved from SQL Server Profiler:
```SQL
SELECT dbo."datetest"."name" FROM dbo."datetest" WHERE dbo."datetest"."created" > TIMESTAMP '2015-01-01 00:00:00'
```